ssl/ssl_lib.c

```diff
@@ -2185,6 +2222,8 @@ long SSL_ctrl(SSL *s, int cmd, long larg, void *parg)
     case SSL_CTRL_SET_MAX_SEND_FRAGMENT:
         if (larg < 512 || larg > SSL3_RT_MAX_PLAIN_LENGTH)
             return 0;
+        if (s->wbio && BIO_get_ktls(s->wbio))
```

Comment:
This restriction should be documented somewhere. Theoretically this could be a breaking change.